keystore/src/connection/platform/generic/mod.rs

```diff
@@ -98,21 +98,72 @@ impl SqlCipherConnection {
     #[cfg(feature = "ios-wal-compat")]
     fn handle_ios_wal_compat(conn: &rusqlite::Connection, path: &str) -> CryptoKeystoreResult<()> {
         const ERR_SEC_ITEM_NOT_FOUND: i32 = -25300;
+        const LEGACY_ACCT_NAME: &str = "keystore_salt";
+        const WIRE_SERVICE_NAME: &str = "wire.com";
         use security_framework::passwords as ios_keychain;
-
         use sha2::Digest as _;
 
+        #[allow(non_upper_case_globals)]
+        fn store_salt(key: &str, salt: &[u8]) -> security_framework::base::Result<()> {
```

Comment:
You could just add a comment on this method for us to remember later for which context it was created